GhostNetwork.Content/Comments/Comment.cs

```diff
@@ -30,5 +31,16 @@ public static Comment New(string text, string key, string replyId, UserInfo auth
         {
             return new(default, text, DateTimeOffset.UtcNow, key, replyId, author);
         }
+
+        public override bool Equals(object obj)
```

Comment:
Don't see any reason for this method

Reply:
Thank you, I forgot about this method, it was needed when the controller returned the comment, and I compared it in unit-tests